pkg/oci/containerize.go

```diff
@@ -150,27 +151,26 @@ func newDataTarball(source, target string, ignored []string, verbose bool) error
 			}
 		}
 
-		header, err := tar.FileInfoHeader(info, info.Name())
-		if err != nil {
-			return err
+		var link string
+		if info.Mode()&os.ModeSymlink == os.ModeSymlink {
+			if link, err = os.Readlink(path); err != nil {
```

Comment:
Could you check that the link does not escape project's root?

Reply:
Sure thing.  I will add a check